tests/main.sh

```diff
@@ -79,6 +79,8 @@ RUNTEST() {
 	fi
 }
 
+TESTCASE"Recursive Walk"
+for i in `seq 1 100`; ls -al "${LXCFSDIR}"; done
```

Comment:
Bit of a lie as it's not actually recursive. Also, we like our tests to be quiet unless they fail.

Reply:
Ah, forgot the R

Reply:
fixed

Reply:
not seeing a fix so either you're still working on the test or you forgot to push :)